parts/common/components.json

```diff
@@ -512,6 +516,14 @@
         {
           "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/azure-cloud-node-manager",
           "latestVersion": "v1.31.2"
+        },
+        {
+          "renovateTag": "registry=https://mcr.microsoft.com, name=oss/kubernetes/azure-cloud-node-manager",
+          "latestVersion": "v1.32.2"
```

Comment:
I've kicked off the windows VHD pipelines to get better logs (they seemed to fail due to quota, which we should have fixed). However, adding two new container images to the windows VHD is very likely going to run out of disk space. Our VHDs are super limited in size and as the size is a product feature (we tell customers the minimum VHD size they need for windows nodes), changing the max size of our VHDs is a product decision. The normal course of action is to to drop some older versions of cloud-node-manager off the VHD when we add more.

Reply:
The windows VHD pipeline passed, which means this hasn't caused us to go over the VHD size limit.